packages/fuselage/src/components/Card/CardControls.tsx

```diff
@@ -1,5 +1,12 @@
-const CardControls = ({ ...props }) => (
-  <div className='rcx-card__controls' {...props} />
+import type { ReactNode, MouseEventHandler } from 'react';
+
+type CardControlsProps = {
+  children?: ReactNode;
+  onClick?: MouseEventHandler<HTMLDivElement>;
+};
+
+const CardControls = ({ onClick, children }: CardControlsProps) => (
+  <div className='rcx-card__controls' onClick={onClick} children={children} />
```

Comment:
Why change this component though? `props` were already being spread...
I would only add the prop types:

```suggestion
import { AllHTMLAttributes } from 'react';

const CardControls = (props: AllHTMLAttributes<HTMLDivElement>) => (
  <div className='rcx-card__controls' {...props} />
```

Reply:
This was my mistake 😅 . I thought it was the standard to use only the necessary props and avoid spreading props like this, but if the default should be the other way around no worries from me.

Reply:
Yes I also thought the same that we require only necessary props when Henrique proposed the change. But nevermind if you want we can go the other way around.